configuration/configuration_test.go

```diff
@@ -0,0 +1,831 @@
+package configuration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"testing"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"strings"
+
+	"github.com/almighty/almighty-core/configuration"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/spf13/cast"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	tDefaultConfigurationFile           = "config.yaml"
+	tEnvironmentVariableNameConfigFile  = "ALMIGHTY_CONFIG_FILE_PATH"
+	tEnvironmentVariableValueConfigFile = "../config.yaml"
+	tInvalidConfigFilePath              = "../invalid_config.yaml"
+	tTestConfigFilePath                 = "../test/data/multiple_config.yaml"
+
+	// The following will be used to set env variables for testing.
+	tTestEnvString         = "ALMIGHTH_CONFIG_VALUE"
+	tTestEnvInt64    int64 = 12345
+	tTestEnvInt      int   = 12
+	tTestEnvDuration       = "1s"
+	tTestEnvBool           = false
+)
+
+func TestGetDefaultConfigurationFile(t *testing.T) {
+	t.Parallel()
```

Comment:
Thanks a lot @kwk - fixed in https://github.com/almighty/almighty-core/pull/712/commits/22fa3db8facccf5a8003de4fa72686ea50685de3